CatAnalyzer/plugins/ttbbLepJetsAnalyzer.cc

```diff
@@ -677,7 +1039,8 @@ void ttbbLepJetsAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetu
   AnalysisHelper trigHelper = AnalysisHelper(triggerNames, triggerBits, triggerObjects);
 
   if ( (ch_tag == 0 && (trigHelper.triggerFired("HLT_IsoMu20_v") || trigHelper.triggerFired("HLT_IsoTkMu20_v"))) ||
-       (ch_tag == 1 && (trigHelper.triggerFired("HLT_Ele22_eta2p1_WP75_Gsf_v") || trigHelper.triggerFired("HLT_Ele22_eta2p1_WPLoose_Gsf_v"))) ) {
+       (ch_tag == 1 ) ) { // TO BE UPDATED: Add electron triggers
+    //(ch_tag == 1 && (trigHelper.triggerFired("HLT_Ele22_eta2p1_WP75_Gsf_v") || trigHelper.triggerFired("HLT_Ele22_eta2p1_WPLoose_Gsf_v"))) ) {
     EvTrigger = true;
   }
```

Comment:
Do we need hard coding for trigger? trigger menu is one of things that can not be fixed throughout the analysis. We can use the HLT filter configuration so that it can be configurable outside and you don't need to compile due to trigger menu changed?

import FWCore.ParameterSet.Config as cms
from HLTrigger.HLTfilters.hltHighLevel_cfi import *
hltHighLevel = hltHighLevel.clone()
hltHighLevel.TriggerResultsTag = cms.InputTag("TriggerResults","","HLT")
hltHighLevel.HLTPaths = cms.vstring('HLT_IsoMu20_v*','HLT_IsoTkMu20_v*')
hltHighLevel.throw = cms.bool(False)